source/what-is-the-bi-connector.txt

```diff
@@ -113,34 +114,34 @@ configure your |jdbc| or |odbc| driver.
    :figwidth: 700px
    :align: center
 
-Hosted Database and On-Premises |bi-short|
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Hosted Database and Self-Managed |bi-short|
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In this configuration, your MongoDB database runs on any 
-internet-accessible host, including any tier of {+atlas+} cluster, and 
-your |bi-short| runs locally.
+In this configuration, your database runs on any tier of {+atlas+} 
+cluster and you install and manage the |bi-short| locally or on your 
+network.
```

Comment:
I don't understand what that means? I understand we now introduce "hosted" vs "locally", but it is not clear what we mean when we say "on your network", in addition to "locally". I hope this a q for tech reviewers to clarify.

Reply:
Thanks for pointing that out, 'network' is a bit of a catch-all. You could have a BI connector installed in a number of places, including in your own AWS EC2 for example. In those cases it'd be "self-managed" but not really local. 

I'll clarify with tech reviewers.